requirements.txt

```diff
@@ -0,0 +1,4 @@
+arrow
+jsonfield==1.0.3
+mbq.metrics==0.4.1
```

Comment:
This is a library so we should be more liberal with the versions, no? `mbq.metrics` is already up to `0.4.3`

Reply:
Same comment in `requirements-dev` with `celery`